Tools/MobileMiniBrowser/Configurations/MobileMiniBrowser.xcconfig

```diff
@@ -21,6 +21,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
 
+OTHER_LDFLAGS = -framework UIKit;
```

Comment:
Would it be better to add UIKit.framework to the project?

Reply:
I don't think we have a preferred style, but I don't mind doing this in the UI.

Reply:
I think that having frameworks in the UI reduces confusion for people who have the project open in Xcode.